compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/Utils.java

```diff
@@ -70,10 +71,22 @@ public final class Utils {
     public static final String FILE_UPLOAD_IDENTIFIER = "Upload";
     public static final String SERVICE_CONFIG_IDENTIFIER = "ServiceConfig";
     public static final String SUBGRAPH_ANNOTATION_NAME = "Subgraph";
+    public static final String UUID_RECORD_NAME = "Uuid";
+    private static final String ORG_NAME = "ballerina";
+    private static final String GRAPHQL_MODULE_NAME = "graphql";
+    private static final String UUID_MODULE_NAME = "uuid";
 
     private Utils() {
     }
 
+    public static boolean isValidGraphQlModule(ModuleSymbol module) {
+        return module.id().orgName().equals(ORG_NAME) && module.id().moduleName().equals(GRAPHQL_MODULE_NAME);
+    }
```

Comment:
Can't we use isGraphqlModuleSymbol method from commons Utils here?

Reply:
Fixed in 9f77931a97791e0655c36447adbe0061df974c7a